schemas/metadata/raw/raw.1.schema.json

```diff
@@ -3,53 +3,69 @@
   "id": "http://jsonschema.net",
   "properties": {
     "args": {
+      "description": "query parameter part of the submission URI, i.e. \"v=4\"",
       "type": "string"
     },
     "content_length": {
+      "description": "Content-Length header",
       "type": "string"
     },
     "date": {
+      "description": "Date header",
       "type": "string"
     },
     "dnt": {
+      "description": "DNT header",
       "type": "string"
     },
     "host": {
+      "description": "Hostname part of the submission URI, i.e. \"incoming.telemetry.mozilla.org\"",
       "type": "string"
     },
     "method": {
+      "description": "HTTP method, i.e. \"POST\"",
       "type": "string"
     },
     "payload": {
+      "description": "Gzip-compressed JSON payload",
       "format": "bytes",
       "type": "string"
     },
     "protocol": {
+      "description": "i.e. \"HTTP/1.1\"",
       "type": "string"
     },
     "remote_addr": {
+      "description": "Source IP address, usually a load balancer",
       "type": "string"
     },
     "submission_timestamp": {
+      "description": "Time when the ingestion edge server in GCP accepted this message; format is ISO 8601 with microseconds and timezone \"Z\", example: \"2018-03-12T21:02:18.123456Z\"",
       "format": "date-time",
       "type": "string"
     },
     "uri": {
+      "description": "i.e. \"/submit/telemetry/6c49ec73-4350-45a0-9c8a-6c8f5aded0cf/main/Firefox/58.0.2/release/20180206200532\"",
       "type": "string"
     },
     "user_agent": {
+      "description": "User-Agent header",
       "type": "string"
     },
     "x_debug_id": {
+      "description": "X-Debug-Id header",
       "type": "string"
     },
     "x_forwarded_for": {
+      "description": "X-Forwarded-For header containing intermediate IP addresses",
       "type": "string"
     },
     "x_pingsender_version": {
+      "description": "X-Pingsender-Version header",
       "type": "string"
     },
     "x_pipeline_proxy": {
+      "description": "Time that the AWS to GCP tee service received the message",
```

Comment:
Or any non-null value (usually ["data-noclip"](https://github.com/mozilla-services/cloudops-deployment/blob/master/projects/data-noclip/puppet/modules/noclip/templates/http_app.erb#L22)) to indicate this data was proxied from AWS. Previously it was populated with the heka timestamp in the same timestamp format as submission_timestamp but there's no more heka. I grepped gcp-ingestion and it doesn't appear we document this anywhere.